src/nidigital/metadata/functions.py

```diff
@@ -971,43 +1316,64 @@
     'FetchCaptureWaveformU32': {
         'codegen_method': 'library-only',
         'documentation': {
-            'description': 'TBD'
+            'description': 'Fetches a defined number of samples for a specific list of sites. This function only returns data from sites that are enabled when fetch is called.\n'
         },
         'parameters': [
             {
                 'direction': 'in',
+                'documentation': {
+                    'description': 'The specified instrument session the niDigital_init or niDigital_InitWithOptions function returns.\n'
+                },
                 'name': 'vi',
                 'type': 'ViSession'
             },
             {
                 'direction': 'in',
                 'is_repeated_capability': True,
                 'repeated_capability_type': 'sites',
+                'documentation': {
+                    'description': 'Site numbers listed as a comma-delimited list of strings of form site\\ ``N``, where ``N`` is the site number. If you enter an empty string, the function fetches data from all sites.\n'
+                },
                 'name': 'siteList',
                 'type': 'ViConstString'
             },
             {
                 'direction': 'in',
+                'documentation': {
+                    'description': 'Waveform name you create with the create capture waveform function. Use the **waveformName** parameter with capture_start opcode in your pattern.\n'
```

Comment:
This is missing from public `fetch_capture_waveform` method's docstring

Reply:
Done